config/initializers/doorkeeper.rb

```diff
@@ -24,8 +24,9 @@ def self.optional
   end
 
   resource_owner_from_credentials do
-    if params[:display_name] && u = User.find_for_database_authentication(display_name: params[:display_name])
-      valid_non_disabled_user = u.valid_password?(params[:password]) && !u.disabled?
+    if params[:display_name] && u = ZooniverseUser.authenticate(params[:display_name],
```

Comment:
Wouldn't they have signed in via devise routes here and thus have a User account? So no need to auth against the Zoo Home account? 

Or would the `u.valid_password?` method be using the wrong hashing alg?